common/djangoapps/config_models/models.py

```diff
@@ -136,9 +145,9 @@ def current(cls, *args):
         return current
 
     @classmethod
-    def is_enabled(cls):
+    def is_enabled(cls, *args):
```

Comment:
@bdero Ok.

Another alternative would be to create another subclass, like `SiteDependentConfigurationModel`, but that might be an overkill for this use case. So I'm fine with what we have now.